test/elixir_console_web/features/console_test.exs

```diff
@@ -41,4 +41,17 @@ defmodule ElixirConsoleWeb.ConsoleTest do
       |> assert
     end)
   end
+
+  feature "visitor can shortcut used expressions", %{session: session} do
+    session
+    |> visit("/")
+    |> fill_in(css("#commandInput"), with: "a = 1 + 2")
+    |> send_keys([:enter])
+    |> send_keys([:up_arrow])
```

Comment:
How about asserting that the commandInput is cleared after `enter`? So we're sure that the 2nd assert clearly means that the value was put there after the up_arrow event.

Reply:
We didn't do that because that was checked in the first feature test.
<img width="527" alt="Screen Shot 2021-09-10 at 12 39 50" src="https://user-images.githubusercontent.com/57733384/132880271-43cbfbb6-a831-4f62-acbb-f6889ad96876.png">

Do you think it is necessary to assert that again in this case?

Reply:
I see, it's not strictly necessary but I think it shows better the flow we're testing. Sometimes a bit of duplication is not a terrible thing.